x-pack/plugins/apm/common/alert_types.ts

```diff
@@ -190,7 +190,7 @@ export const ALERT_TYPES_CONFIG: Record<
   },
   [AlertType.TransactionDurationAnomaly]: {
     name: i18n.translate('xpack.apm.transactionDurationAnomalyAlert.name', {
-      defaultMessage: 'Latency anomaly',
+      defaultMessage: 'Anomaly',
```

Comment:
`TransactionDurationAnomaly` type and  `'xpack.apm.transactionDurationAnomalyAlert.name'` can be renamed as it now refers to all metrics